tsdb/tsdbutil/chunks.go

```diff
@@ -18,23 +18,37 @@ import (
 	"github.com/prometheus/prometheus/tsdb/chunks"
 )
 
+type Samples interface {
```

Comment:
Why are you adding this? Using a slice directly would be simpler.

Reply:
Hm hm, I copied it over